### ISSUE TITLE
child jenkins slave base images are not triggering if new build of jenkins slave base image

### ISSUE THREAD

**Issue by gerardcl** [opendevstack/ods-project-quickstarters#118]
it would be great that if a new build of the Jenkins slave base image is happening then its child images (slave images) are triggered to be rebuild again too.

**Comment by clemensutschig** [comment]
@gerardcl - all slaves have the base slave set as FROM ... this does not trigger a rebuild automatically (on DC level you can set this as image/change trigger) ... whats your thought on this?

**Comment by gerardcl** [comment]
exactly, that is the idea, so to change the DC template. is this ok?

**Comment by gerardcl** [comment]
indeed there is  also the chance to load a trigger in the build config, which is what we want indeed: 
```
spec:
  triggers:
    - imageChage:
      type: ImageChange
```

**Comment by gerardcl** [comment]
ok, found it:
``
oc set triggers bc <build-name> --from-image='<image-name/upstream-image-name>' -n <project-name>
```
